### PR TITLE
Some more cleanup on services.adoc

### PR DESCRIPTION
**Pull request by ignasi35**
While cleaning up TODO into (1) WONT_DO or (2) new issues I stumbled upon `services.adoc`.

It's almost there, but there are two or three TODO's that need some attention. I'm creating this PR as DRAFT to try to iron them down and finish the page.

### CODE REVIEW

**Comment by ignasi35** [comment]
> It's almost there, but there are two or three TODO's that need some attention. I'm creating this PR as DRAFT to try to iron them down and finish the page.

These TODOs are not a blocker. We can fix them in the future.

File: docs-source/docs/modules/concepts/pages/services.adoc [hunk 1]
```diff
@@ -5,39 +5,31 @@
...
 
 To communicate the API consistently, we recommend using schema-first approaches. The schemas are to be shared by the service implementation and its clients. From the schema the implementing code is generated and used to implement the service and the client code. With this approach the service, and the clients don't share any code or dependencies.
 
 == Synchronous Services
 
-This guide illustrates building service APIs for synchronous communication for use cases which require an instant reply. Most often synchronous communication issues a single request and expects a single reply. Other cases open communication channels for continued communication with possibly ongoing replies.
+In the first steps of the xref:microservices-tutorial:index.adoc[Implementing Microservices with Akka tutorial], the tutorial illustrates building service APIs for synchronous communication for use cases which require an instant reply. Most often synchronous communication send a single request and expects a single reply. Other cases open communication channels for continued communication with possibly ongoing replies.
```
Comment (patriknw) [inline docs-source/docs/modules/concepts/pages/services.adoc:15]: "with Akka tutorial, the tutorial"

repeating tutorial might look funny. 

"the first steps of the Implementing Microservices with Akka tutorial illustrates ..." ?

File: docs-source/docs/modules/concepts/pages/services.adoc [hunk 2]
```diff
@@ -68,6 +60,5 @@ ifdef::todo[TODO: Service descriptor agnostic to transport?]
 
 A Service may experience transient failure or degradation due to excessive load, GC pauses, etc... in that case, clients will retry the failed request, which will increase the load. Instead, in order to let the service recover from the transient failure as fast as possible, clients should use xref:circuit-breakers.adoc[Circuit Breakers].
 
-== Structuring our Service
-
-ifdef::todo[Anti-corruption layer]
+ifdef::todo[== Structuring our Service]
+ifdef::todo[Anti-corruption layer]
```
Comment (patriknw) [inline docs-source/docs/modules/concepts/pages/services.adoc:64]: when rendered in author mode there is no indication that this is a TODO

File: docs-source/docs/modules/concepts/pages/services.adoc (hunk 1 above)
Comment (ignasi35) [reply docs-source/docs/modules/concepts/pages/services.adoc:15]: You're right.

Comment (ignasi35) [reply docs-source/docs/modules/concepts/pages/services.adoc:15]:
```suggestion
The first steps of the xref:microservices-tutorial:index.adoc[Implementing Microservices with Akka tutorial] illustrate building service APIs for synchronous communication for use cases which require an instant reply. Most often, synchronous communication sends a single request and expects a single reply. Other cases open communication channels for continued communication with possibly ongoing replies.
```

File: docs-source/docs/modules/concepts/pages/services.adoc [hunk 3]
```diff
@@ -5,39 +5,31 @@
...
 
-This guide illustrates building service APIs for synchronous communication for use cases which require an instant reply. Most often synchronous communication issues a single request and expects a single reply. Other cases open communication channels for continued communication with possibly ongoing replies.
+The first steps of the xref:microservices-tutorial:index.adoc[Implementing Microservices with Akka tutorial] illustrate building service APIs for synchronous communication for use cases which require an instant reply. Most often, synchronous communication sends a single request and expects a single reply. Other cases open communication channels for continued communication with possibly ongoing replies.
 
 Synchronous communication gives results and errors immediately so the client can react before continuing its processing, but that often means the client doesn't progress until it received the response.
 
 === gRPC
 
-gRPC services use Protobuf as their language-agnostic schema and which translate to efficient serialization.
+xref:grpc-services.adoc[gRPC services] use Protobuf as their language-agnostic schema and which translate to efficient serialization.
```
Comment (octonato) [inline docs-source/docs/modules/concepts/pages/services.adoc:21]:
```suggestion
xref:grpc-services.adoc[gRPC services] use Protobuf as their language-agnostic schema which translates to efficient serialization.
```

What translates to efficient serialization if the fact that it's Protobuf, hence the usage of third person.

**Review by octonato — APPROVED** [review]
LGTM

Minor suggestion on text that was already there, but better to change since we are already touching it.

**Pull request merged**